src/nemo-window-menus.c

```diff
@@ -1786,6 +1787,19 @@ nemo_window_initialize_menus (NemoWindow *window)
 	gtk_action_group_add_actions (action_group,
 				      main_entries, G_N_ELEMENTS (main_entries),
 				      window);
+
+      /* if root then hide menu items that do not work */
+      if (geteuid() == 0) {
+          action_to_hide = gtk_action_group_get_action(action_group, "Go to Computer");
+          gtk_action_set_visible(action_to_hide, 0);
```

Comment:
use FALSE not 0 for these

Also always put spaces between the function names and parentheses:
```
gtk_action_set_visible (action_to_hide, FALSE);
```

Reply:
Changed 0 to FALSE and added spaces.